lib/adapters/parsers/csv.js

```diff
@@ -33,7 +33,7 @@ class CSVParser extends base {
                     self.doneParsing = true;
                     emit(buffer);
                     buffer = [];
-                    csvStream.end();
+                    csvStream.emit('end');
```

Comment:
@davidvgalbraith yes this is necessary now since `fast-csv` doesn't really care if you call the `end()` method and only responds to the `end` event. Figured this out through some code inspection and trial and error.